package.json

```diff
@@ -12,33 +12,33 @@
         "preset": "jest-expo"
     },
     "dependencies": {
-        "@expo/vector-icons": "^10.0.0",
+        "@expo/vector-icons": "^12.0.0",
         "@react-native-community/masked-view": "0.1.10",
-        "@react-navigation/bottom-tabs": "^5.0.0",
-        "@react-navigation/native": "^5.0.0",
-        "@react-navigation/stack": "^5.0.0",
-        "@react-navigation/web": "~1.0.0-alpha.9",
-        "expo": "^39.0.0",
-        "expo-asset": "~8.2.0",
-        "expo-constants": "~9.2.0",
-        "expo-font": "~8.3.0",
-        "expo-linking": "^2.3.1",
-        "expo-splash-screen": "~0.6.2",
-        "expo-web-browser": "~8.5.0",
+        "@react-navigation/bottom-tabs": "^5.10.0",
+        "@react-navigation/native": "^5.10.0",
+        "@react-navigation/stack": "^5.10.0",
+        "@react-navigation/web": "~1.10.0-alpha.9",
+        "expo": "^42.0.0",
```

Comment:
why is an expo upgrade part of this PR?

Reply:
I really do not know how this is part of this PR . expo update is now merged in master and before was on a separate branch which was the base for this branch .  I can see that , the first 3 commits in this PR actually belongs to the expo update branch , reason could be that I have done a reset and not rebase ? 

How I would try to clean it is : doing a cherry-pick of the commits that belongs to  books-API-request branch , into a fresh created branch based on master and then I would need to rebase my other branch for the book list exercise on this last fresh created branch , because they belong to each other. 
Finally I'll be able to delete this books-API-request branch . 

Asking myself if it's all worth it , your opinion on this matter will be decisive

Reply:
`reason could be that I have done a reset and not rebase` what did you reset to what?

Reply:
i don't quite know why the 3 commits were shown in this PR as the commits (even with the same commit hashs) where already in master. usually they do not show up in the diff then.

maybe it was due to resetting, but i don't think so.

it was probably because of you targeting master with this PR, when the sdk pr was not merged yet, and github did not update the diff with the master after that one was merged.

if you had targeted this branch on the sdk branch, the diff would not have included the commits in the beginning, and github would have retargeted this PR to master, once the other branch was merged into master. (!! CAREFUL: GITLAB MIGHT HANDLE THIS DIFFERENTLY !!)

anyways. your suggestion on how to resolve this issue would have been a lot of hassle, and would separate all the discussions/PR from the branch etc.

since they had the same commit hashs, there was no need for drastic measures. it might have been enough to simply change target back and forth to master, but i did a rebase onto master, which updated the diff as well (and added the merge commit of merging sdk updates into master to the branch). nothing else changed though.

Reply:
also: please always think, if it is really necessary, when you reset branches. resetting, rebasing and amending commits needs to be done thoughtfully, because it alters the history of changes.
it might make them look better, keep them more readable etc, but it can cause problems if done incorrectly.